sample/F8/event

```diff
@@ -0,0 +1,54 @@
+{
+	"id": 180,
+	"name": "F8-Facebook Developer Conference 2017",
+	"latitude": 37.329008,
+	"longitude": -121.888794,
+	"location_name": "San Jose Convention Center, 150 W San Carlos St  San Jose  CA USA 95113 ",
+	"start_time": "2017-04-18T10:00:00",
+	"end_time": "2017-04-19T17:00:00",
+	"timezone": "US / Pacific",
+	"description": "Join us for our annual 2-day event, where developers and businesses come together to explore the future of technology. Learn how Facebook connects the world through new products and innovation. This year’s event is bigger than ever – with more than 50 sessions, interactive experiences, and the opportunity to meet one-on-one with members of the Facebook team.",
+	"background_image": "/imagesbackground.png",
```

Comment:
@iamareebjamal i am not blaming you at all.
but if i am adding 200+ sessions a small mistake can happen. so help might be needed in such cases because you get the web app but not the android app.